### PR TITLE
Unify metronome build script

### PR DESCRIPTION
**Pull request by janisz**
## High-level description

Use the same build script in OSS and EE. This allows us to remove metronome package from EE since it will differ only on configuration.

## Corresponding DC/OS tickets (required)

  - [D2IQ-ID](https://jira.d2iq.com/browse/D2IQ-ID) JIRA title / short description.

## Related tickets (optional)

  - [D2IQ-ID](https://jira.mesosphere.com/browse/D2IQ-<number>) JIRA title / short description.

## Checklist for component/package updates:

If you are changing components or packages in DC/OS (e.g. you are bumping the sha or ref of anything underneath `packages`), then in addition to the above please also include:

  - [ ] Change log from the last version integrated (this should be a link to commits for easy verification and review): [example](https://github.com/dcos/dcos-mesos-modules/compare/f6fa27d7c40f4207ba3bb2274e2cfe79b62a395a...6660b90fbbf69a15ef46d0184e36755881d6a5ae)
  - [ ] Included a test which will fail if code is reverted but test is not. If there is no test please explain.
  - [ ] Test Results: [link to CI job test results for component]
  - [ ] Code Coverage (if available): [link to code coverage report]

### CODE REVIEW

**Comment by janisz** [comment]
@mesosphere-mergebot bump-ee

**Comment by janisz** [comment]
@mesosphere-mergebot request-review @jeschkies @timcharper

**Comment by janisz** [comment]
@mesosphere-mergebot label Ready For Review

**Comment by janisz** [comment]
@mesosphere-mergebot request-review @jeschkies @timcharper

File: packages/metronome/build [hunk 1]
```diff
@@ -21,7 +21,9 @@ PermissionsStartOnly=True
 EnvironmentFile=/opt/mesosphere/etc/check_time.env
 EnvironmentFile=/opt/mesosphere/environment
 EnvironmentFile=/opt/mesosphere/etc/metronome
+EnvironmentFile=-/opt/mesosphere/etc/metronome-extras
```
Comment (jeschkies) [inline packages/metronome/build]: What is in `metronome-extra`?

Comment (janisz) [reply packages/metronome/build]: nothing, this file does not exist in OSS. It's filled only in EE. We use this approach in other packages as well 
```ini
# git grep "EnvironmentFile=.*-extras"
packages/exhibitor/extra/dcos-exhibitor.service:EnvironmentFile=-/opt/mesosphere/etc/exhibitor-extras
packages/marathon/build:EnvironmentFile=-/opt/mesosphere/etc/marathon-extras
packages/mesos-dns/extra/dcos-mesos-dns.service:EnvironmentFile=-/opt/mesosphere/etc/mesos-dns-extras.env
packages/mesos/extra/dcos-mesos-master.service:EnvironmentFile=-/opt/mesosphere/etc/mesos-master-extras
packages/mesos/extra/dcos-mesos-slave-public.service:EnvironmentFile=-/opt/mesosphere/etc/mesos-slave-common-extras
packages/mesos/extra/dcos-mesos-slave.service:EnvironmentFile=-/opt/mesosphere/etc/mesos-slave-common-extras

```

**Comment by jeschkies** [comment]
@mesosphere-mergebot changelog-not-required internal change.

**Comment by janisz** [comment]
@mesosphere-mergebot label Ready For Review

**Comment by janisz** [comment]
@mesosphere-mergebot request-review @jeschkies @timcharper

**Comment by janisz** [comment]
@mesosphere-mergebot label Ready For Review

**Pull request merged**